corehq/apps/hqcase/tests/test_case_list_api.py

```diff
@@ -108,6 +108,7 @@ def tearDownClass(cls):
     ("closed=true", ["ned"]),
     ("date_opened.lt=1878-02-19", ["mattie", "rooster"]),
     ("date_opened.lte=1878-02-19", ["mattie", "rooster", "laboeuf"]),
+    ("date_opened.eq=1878-02-19", ["laboeuf"]),
```

Comment:
does this only accept dates or does it also accept date times (or other partially truncated date time values)

Reply:
hmm, that's a good point.  It accepts dates and datetimes, but I think the datetime behavior is probably bad.  It appears to coerce to either seconds or microseconds:
```python
>>> parse("2020-03-08T12:53").isoformat()
'2020-03-08T12:53:00'
>>> parse("2020-03-08T12:53:01.3").isoformat()
'2020-03-08T12:53:01.300000'
```
That might still be useful as long as the behavior is well explained (ie, that you can't use arbitrary precision).  I could also see restricting the `eq` filter to just dates for simplicity's sake.  I'll do some fiddling in tests in a follow-up PR.

Reply:
Actually, my `.eq` test is failing in Travis, but it passes locally - I think I'll revert that stuff for now and work on it in isolation separately.  It feels kind of fragile - might not be worth the headache.